### PR TITLE
Xapid 1049 apid updates for Org/Env scopes

### PR DESCRIPTION
**Pull request by haomingzhang**
apid updates for Org/Env scopes

### CODE REVIEW

File: data.go [hunk 1]
```diff
@@ -371,18 +371,28 @@ func findScopesForId(configId string) (scopes []string) {
 
 	log.Debugf("findScopesForId: %s", configId)
 
-	var scope string
+	var scope sql.NullString
 	db := getDB()
 
-	rows, err := db.Query("select DISTINCT scope from EDGEX_DATA_SCOPE where apid_cluster_id = $1", configId)
+	query := `
+		SELECT scope FROM edgex_data_scope WHERE apid_cluster_id = $1
```
Comment (sramamoorthy73) [inline data.go]: UNION ALL is always more performant than plain UNION, as along as you know that two SELECTs are already producing distinct results.

File: listener_test.go [hunk 2]
```diff
@@ -144,11 +146,11 @@ var _ = Describe("listener", func() {
 			Expect(ds.Scope).To(Equal("s2"))
 
 			scopes := findScopesForId("a")
-			Expect(len(scopes)).To(Equal(2))
-			Expect(scopes[0]).To(Equal("s1"))
-			Expect(scopes[1]).To(Equal("s2"))
-
-			//restore the last snapshot
+			Expect(len(scopes)).To(Equal(6))
+			expectedScopes := []string{"s1", "s2", "org_scope_1", "env_scope_1", "env_scope_2", "env_scope_3"}
```
Comment (sramamoorthy73) [inline listener_test.go]: Ok, so you assuming that the values will be prefixed with the scope type. Is that being done in the apid mgmt?

File: data.go (hunk 1 above)
Comment (haomingzhang) [inline data.go]: Yes, I agree that UNION ALL is more performant. Which two SELECTs are already producing distinct results?

File: listener_test.go (hunk 2 above)
Comment (haomingzhang) [inline listener_test.go]: That's a good point. Not done in mgmt. What I assume is not about prefix. What I assume is org_scope, scope and env_scope cannot be same, which means the "UNION" will never deduplicate an org_scope, an env_scope and a scope.

**Pull request merged**